Mathlib/CategoryTheory/Adjunction/Evaluation.lean

```diff
@@ -93,26 +84,14 @@ def evaluationRightAdjoint (c : C) : D ⥤ C ⥤ D where
   obj d :=
     { obj := fun t => ∏ᶜ fun _ : t ⟶ c => d
       map := fun f => Pi.lift fun g => Pi.π _ <| f ≫ g }
-  map f :=
-    { app := fun _ => Pi.lift fun g => Pi.π _ g ≫ f
-      naturality := by
-        intros
-        dsimp
-        ext
-        simp }
+  map f := { app := fun _ => Pi.lift fun g => Pi.π _ g ≫ f }
 
 /-- The adjunction showing that evaluation is a left adjoint. -/
 @[simps! unit_app_app counit_app]
 def evaluationAdjunctionLeft (c : C) : (evaluation _ _).obj c ⊣ evaluationRightAdjoint D c :=
   Adjunction.mkOfHomEquiv
     { homEquiv := fun F d =>
-        { toFun := fun f =>
```

Comment:
Just to make sure: you eyeballed that performance is fine here, right?

Reply:
Yep, thanks for checking